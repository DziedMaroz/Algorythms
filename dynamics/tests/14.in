9607832736765
737
7017
38
016
7919122
5
6898299143755
1817235
8485
360
9441763
966751
09692416678
015985319643
722091832125
1146896845387
4219926
170761
1853928
568335128653
746
72
024425
3486676239038
796588
532558
4296112423481
516
308179
543144
8147
5367
73
1178083
1979557640308
50
0557837
66
7
511846
7258919
14979723102
54
1
73020657171
98573
319888
5089969434
67616
59071331
496933
47235
06403871948
921880410746
61592
38365
8597385484
121627508
2316
73445
08127558828
57520
217891934653
1630597
87
169140631458
322
838036686352
299826
8
846
47707888
179462949626
3688583794
59184
58552935161
37769704573
9487
8691489
45
42091
4362891
1749154695
7492251673227
558024320
15148
225823522216
474
5
5492199
4366329
02602
86599
226642073684
3576507
96771
128868036375
57446417
0535532
3148461
0591253
434
5188265228842
865077864138
7333661
766
2
9971
819312
06776
70145551
712280
5247885420
5669110603450
99925
466219033
68707
366
45
37
1
4426697176008
7222349
08497592
29439
9824619
31
1
907253962942
24299789
44075276089
3
0321786
816339511046
9
0
93
7503
7229
22
98
8879889564
781635
466430503
581510261637
821111070
147690101661
555
95546491014
3948215504688
5952804022610
6177936026
9441214
58724230595
829029826
738625826115
3332
4712
8847099500
039460
394849524857
68462948590
17
2
3923316
236161
66999
943012312055
27071661
1
8929
2747361
80526796
406376
143
65505894835
161929
5219248449588
9741157973617
43547
8048
06969540
9046
494992154311
199132
1074699466515
14395
3704360489
47
3886899
493261299153
482339372213
007069406
268
25666
267072521538
96370734
60951505458
38
0889704489
11580612
823059
351027
3832712328
514514
670406326
929974865689
417
8
799
954
3488
780497017802
9901905060
690764980
1
367066087
47126
32
188397
32
28
054782
0
72283
0883825548953
083051734
0356404106603
0259620134
8942782012656
2920609996
4027
1925443
6310
831165144913
2
80409139278
31637365733
33544
07295069
2405831934277
57917
21668947813
059876027120
2407
19320498
35
60853830120
7120263149
708999
061536730148
4058177718
9211
17
958210531022
694541606
834
412505
96933182292
8649582993925
3774390857102
5
5769600259385
873590837810
3
0288
54
86870955
46010756
0158005796
3499305841
97
00
31
13
03359088
1
1387
069665
5476
6070767976589
998845241324
8006004
9123
432688886820
53089459
33
102397227877
94407
808
0380
99502986706
035757368
591660275
9592113803204
8925097853
13
56595115
37143480
4077482
71164010757
36647493
794
492
03901810
2
4
651
796
3366334303385
46062714
545837195
25163760
502
15824
630450197
63862
0
6796346002
6976651691
80
40
620
531946567795
1440
1
04
373160871
865
7761797262933
9920700165999
68010
216807245
08
10
6010668306228
57
932764
0808
81604806
745683284203
920630742
71048720
76732202
996
45
8935943
69
327755261
08
7
804649421
263263511791
85848
9576567224
7744185
1042434115899
245075014226
376174
486739
6926622
524410680
527265891
8329709314
26086
5193934
23414
00842
991185561536
47112604734
7
929672
591875586187
582
06135337
53430
95677718525
27661047320
9762744816
7516
513751
948179154965
2518318281717
3617038406046
4439175
75003741
69
30
0
31134
794
9663704
52
1023
137538
507468
3349244910
860631228
6916242320818
5422977495323
93198
72779541642
31
8151
18651073
630603359
65100104
350
72
2
2
0809478506732
39
06474
3030
9616415875199
6745038801
8821410462
6324246866517
9
7
69
003230281
423102245
5354420215944
414094922
0
189228689192
1468251
3
1723
590547957939
4207618636
2947954
145
77716
1143212663
1890
4341
242278870
022944729952
9625
39142894337
55
51642980
70544271
189487
129707253
3824063943
1375692633024
2259
4478750567
2123
44747423
6263861825362
3421703610
9
433952877
76
492957606
503074
2562477430324
28558113825
33
41584317145
2857206
4421
051141
2227594
69144582901
42
38657025611
02732757567
77
5062
811439
345627
08312068742
7960458000
4106057257
84975781565
356985246214
87
107724490
0164
39
859400512
842235220464
301513
968269
7069
2729
935
637
357042165
533521
9
4719
054
5934
498352126
180420
316680707
829498668
092566
84
5357796701
8357262
3266340
0775
5325384
2446097
562204254923
30253373556
67970709
7462883980018
849853960978
61177
927197525
03578883221
6656441021121
45618
2785710045515
41908703309
71348
21
1070125
18987056327
3
584576663
995943
43017665
1417736220328
6274764
34
2858180
9861729090965
3385127603410
4543024
9
9854223
2
46723
12260006
4
130
987008
2082080
357322
029298425
0001442
91836309
212
071102526180
4903
62700330438
33
16349553
172954549591
322202766839
25790625
939502
39215
1446204
93030316645
76739784
185778236
25559
581778591
86
667421582784
53196059943
7462590531469
416168611233
830824148
438050672
24396492
21014309
094994628370
62
447113
9017779143
583063618
71308
1529095605421
0
4618718
354098
455
029347070660
33114135
09153315
676667405
622166026959
657
36
6001331689
44
175
8034473
9927769167703
1
642711
3063224
65
286237265
88748
8090454755475
0469968
2383
4937798079240
4331070526
9277928281647
2
643079
16143132
56248
709360
88726830
4454026
856
49926959
4203682098385
4675231327
701504253
8675
0382087995326
386795
8646565
9179400240163
8790921731775
73636394
4533922959
17
143
293
4116657154
53810
142102190
9390215358
4122
4839821552
6836742320
451532942998
343313718
5115329551767
8392197338891
6447
167560087
558932
5334359
4337
33899149
461773
75458635
0481202
20286
565188672
8323
283
9844894549431
807108685
1947680
13
049342257
8030225817
587449
81
01889367
0333507
177681881
312844301329
145
7782543868339
8879217
6
8082294833
5504763879
0111942160584
3914405
33
1758257
79752
60813003
18792
4
9574
741042568748
102
89226804
69
7271055965
57891576
22546420667
331
2
292520
642120253
99151
771829
923
4078
25901639629
0903082
091
8
6434464183
40919879372
250
5297701
0874179
456275
610
6272209521
69
1322712799
6269
394825
54437
5327
369897
8496410
05291836602
9749431189
5174546570
5068528
0765
8042918605442
127949
182
3
1
708282
43271260399
76213430456
366211865561
72812
000
520541558
