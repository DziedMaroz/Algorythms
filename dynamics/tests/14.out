10
9 6 0 7 8 32 7 36 7 65
